joomla/joomla-cms
# Protected menutype 'menu' not protected anymore

### Summary of Changes
Menutype ‘menu’ is no longer protected. 
Do not allow batch/trash/delete for protected menu items.

### Testing Instructions
Create a menutype 'menu', you should be able to create it without any warning after this patch. Without this patch a warning will be shown saying it is a reserved name.

Menu manager menutype dropdown filter no longer includes 'menu' as protected, verify.

Delete/Trash/Batch toolbar buttons should not be displayed in menu manager when filtered by the protected menutype 'main'.
 
### Documentation Changes Required
Not sure
